common/utils.js

```diff
@@ -453,6 +458,27 @@ const getItsEdgeContract = (chainConfig) => {
     return itsEdgeContract;
 };
 
+const downloadWasmFile = async (url, contractName, version) => {
```

Comment:
```suggestion
const downloadContractCode = async (url, contractName, version) => {
```
not all VMs use wasm

Reply:
done